inductiva/molecules/scenarios/utils.py

```diff
@@ -0,0 +1,109 @@
+# pylint: disable=protected-access
+"""Post processing auxiliary functions."""
+import time
+import os
+import io
+import base64
+import shutil
+import threading
+import PIL
+from moviepy.editor import ImageSequenceClip
+from ipywidgets import Output, IntProgress
+
+
+class MovieMaker:
+    """ 
+    Class to create a movie from a trajectory rendered on 
+    NGLview.
+    """
```

Comment:
I'm not sure why we need this class at all. All it is doing is looping over frames, saving each individual frame, and then combining the frames into a movie.

This could easily be achieved by having the loop over frames in the output class, and use the function `create_movie_from_frames` in `inductiva/utils/visualization.py` to merge the frames, right?

Also, just checked that `NGLViewer` has a method `download_image` that renders the view and exports it to a file, so all the encoding could be spared?

Reply:
yes, nglview has that method, however, It was causing problems because it was downloading to the downloads folder independently of the path I put, that's why I did it like this! I will try to use the create_movies_from_frames :)